data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+				accept
+	
+	on visit
+		dialog `You do not have the jump drives. You either did not go to <destination> to get the jump drives yet, or left your escort allocated to carrying the jump drives behind, in which case better launch and wait for it.`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	to complete
+		has "Unfettered: Jump Drive Retrieval: offered"
+	on complete
+		outfit "Jump Drive" -4
+		payment 3000000
+		log `Helped a young Unfettered captain to bring jump drives from a pirate world. It was a really... unforgettable experience, I almost died multiple times but in the end I saved him.`
+		log "People" "Si-Ra" `A young impetuous Unfettered captain who seems to grow wiser by the day. He has impressive leading and combat skill, and seems to be a fan of getting bigger and bigger ship. He always leaves room for improving tactics.`
+		dialog `"Here is your share : <payment>. You have saved my life, captain <first>, and I shall never forget it. You have earned the right to know my name : "Si-Ra". Come see me again soon if you are up for a challenge, I am sure that I will have what you want." Then he leaves, after doing what you assume to be a salute to you, his fist risen. You are taken by surprise by the act, and before you have the time to react, he is already leaving. He is probably in a hurry to buy that new ship he told you about.`
+
+
+
+mission "Unfettered: Attracted Attention"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove your strength 2: active"
+	on offer
+		require "Jump Drive" 1
+	on enter "Almaaz"
+		dialog `The Unfettered comes to the bridge to warn you, "There are even more unfettered humans than anticipated. I told you not to come here with a jump drive!"`
+	npc evade
+		government "Pirate"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+mission "Unfettered: Jump Drive Retrieval"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove Your Strength 2: active"
+	on offer
+		conversation
+			`Following the advice of the Hai, you park your ship close to the hidden depot in case something goes wrong, and then start following him to the hatch. He is moving at a quick pace, you have difficulties following him, even given the fact he is wearing a combat suit.`
+			choice
+				`	"Who made this vault, what should I expect?"`
+				`	(No time for talking.)`
+					goto outside
+			`	"That is an information I cannot trust anyone with, not even you, and especially not here. I do not want the owner of this cache to be in danger whilst we need jump drives to expand. Assume it is owned by unfettered humans, or pirates as I think you call them, and that we are ready to face them," he tells you. He is fully armoured, meaning as long as you stay with him, everything should be fine.`
```

Comment:
`He is fully armoured, `**meaning as long as you stay with him, everything should be fine.**
Another weird aside that is out of place.

Reply:
I tried to hint at the player they should not separate from the Hai but maybe he should say it explicitly instead.

Reply:
Or simply let the player work out that staying with the heavily armoured Hai is a smart move.